Client/src/app/components/historyComponent/history.component.less

```diff
@@ -0,0 +1,26 @@
+.history-list {
+  border: 4px solid dimgray;
+  border-radius: 5px;
+  width: 30%;
+  height: 27.7em;
+  cursor: pointer;
+}
+
+.history-title {
+  padding-left: 10%;
+  padding-top: 2.5%;
+  padding-bottom: 0.5%;
+  font-size: 25px;
+}
+
+.button {
```

Comment:
I know it's in the  context of the historyComponent, but still you should give 'button' a better name